lib/experimental/filter_output.dart

```diff
@@ -0,0 +1,11 @@
+final RegExp _flutterUrlExp =
```

Comment:
If these are going to be in a separate file, you'll need to pop the copyright header in at the top.

Reply:
Good point. Moved this code into the existing file instead